client-proxy/database/database.go

```diff
@@ -0,0 +1,45 @@
+package database
+
+import (
+	"database/sql"
+	"github.com/OpenLineage/OpenLineage/client-proxy/config"
+	_ "github.com/mattn/go-sqlite3"
+	"log"
+)
+
+type Database struct {
+	db *sql.DB
+}
+
+type IDatabase interface {
+}
+
+func New(conf config.Config) *Database {
+	db, err := sql.Open("sqlite3", conf.SqliteDatabasePath)
+	if err != nil {
+		log.Fatalln("Cannot open database:", err)
+	}
+
+	log.Println("Initializing DB")
+	InitDB(db)
+	return &Database{db: db}
+}
+
+func InitDB(db *sql.DB) {
+	db.Exec(`
+		CREATE TABLE IF NOT EXISTS partitions (
+			name       text     NOT NULL,
+			size       integer  NOT NULL,
+			created_at datetime NOT NULL,
+			current    boolean  NOT NULL
```

Comment:
_Minor_: maybe rename to `is_current`? as `current` reads more like a pointer

Reply:
yeah, that sounds better.